server/views/layout.njk

```diff
@@ -26,6 +26,22 @@
         });
     {% endif %}
 
+    {% if barcodeImages | length %}
```

Comment:
Do you need a similar block of code for `numberOfCoversheets` ?

Reply:
Or does the number of barcodeImages cover both copy/paste barcode & PDF journeys?

Reply:
Oh yeah! Thanks.